votor/src/certificate_pool.rs

```diff
@@ -87,6 +87,105 @@ impl From<AlpenglowCommitmentError> for AddVoteError {
     }
 }
 
+#[derive(Debug, Default)]
+struct FallbackVoteCounters {
```

Comment:
lets move this to a new file in `votor/src/certificate_pool/`

Reply:
Also thinking we name it something more generic like `SlotStakeCounters`, so in the future we could just aggregate certificates using `notarize_entry_total` (and add a `finalize_total`, notarize fallback, skip fallback etc.) and then we can remove the per type vote pools.